pkg/cli/cli.go

```diff
@@ -446,10 +446,9 @@ func hostCommand(opts *subCommandOpts) *cobra.Command {
 		},
 	}
 	create.Flags().StringVar(&createFlags.GCP.MachineType, gcpMachineTypeFlag,
-		opts.InitialConfig.Host.GCP.MachineType, gcpMachineTypeFlagDesc)
+		opts.InitialConfig.DefaultService().Host.GCP.MachineType, gcpMachineTypeFlagDesc)
 	create.Flags().StringVar(&createFlags.GCP.MinCPUPlatform, gcpMinCPUPlatformFlag,
-		opts.InitialConfig.Host.GCP.MinCPUPlatform, gcpMinCPUPlatformFlagDesc)
-	create.Flags().StringArrayVar(&acceleratorFlagValues, acceleratorFlag, nil, acceleratorFlagDesc)
+		opts.InitialConfig.DefaultService().Host.GCP.MinCPUPlatform, gcpMinCPUPlatformFlagDesc)
```

Comment:
Why do we set the default value as configuration of default service? In case of given service is not a default one, shouldn't other flags be depending on what the service is?